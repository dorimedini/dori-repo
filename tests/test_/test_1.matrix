{
    matrix A(2, 3) = [1, 2, 3; 4, 5, 6];
    print "A =";
    print A;

    matrix B(3, 2) = [1, 2; 3, 4; 5, 6];
    print "B =";
    print B;

    matrix C(2, 2) = A * B;
    print "";
    print "A*B =";
    print C;
}
