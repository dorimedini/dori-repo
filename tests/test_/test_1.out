A =
[1,2,3;4,5,6]
B =
[1,2;3,4;5,6]

A*B =
[22,28;49,64]
